dffml/base.py

```diff
@@ -248,7 +253,12 @@ def config_get(cls, config, above, *path) -> BaseConfig:
             if "nargs" in arg:
                 value = list(map(arg["type"], value))
             else:
-                value = arg["type"](value)
+                # instantiation of generic collections are not allowed
+                try:
+                    value = arg["type"](value)
+                except TypeError:
+                    value = locate(value)
```

Comment:
On second thought. I think this is caused by `arg["type"]` being typing.Type. Maybe instead of a `try / except`, see if an `if arg["type"] == typing.Type: value = locate(value)` works.

Reply:
works :+1: